### PR TITLE
`azurerm_netapp_pool` - update the validation of `size_in_tb`

### PR DESCRIPTION
**Pull request by neil-yechenwei**
fixes https://github.com/hashicorp/terraform-provider-azurerm/issues/22931

image

### CODE REVIEW

File: website/docs/r/netapp_pool.html.markdown [hunk 1]
```diff
@@ -48,7 +48,7 @@ The following arguments are supported:
 
 * `service_level` - (Required) The service level of the file system. Valid values include `Premium`, `Standard`, or `Ultra`. Changing this forces a new resource to be created.
 
-* `size_in_tb` - (Required) Provisioned size of the pool in TB. Value must be between `4` and `500`.
+* `size_in_tb` - (Required) Provisioned size of the pool in TB. Value must be between `2` and `500`.
```
[the remark this answers is not in the document]
Comment (neil-yechenwei) [reply website/docs/r/netapp_pool.html.markdown:51]: Updated.

**Comment by neil-yechenwei** [comment]
@jackofallops , thanks for the comment. I updated PR. Please take another look. Thanks.

File: website/docs/r/netapp_pool.html.markdown [hunk 2]
```diff
@@ -48,7 +48,9 @@ The following arguments are supported:
...
+
+~> **NOTE** `2` TB capacity pool sizing is currently in preview. You can only take advantage of the `2` TB minimum if all the volumes in the capacity pool are using `Standard` network features. If any volume is using `Basic` network features, the minimum size is `4` TB. See [here](https://learn.microsoft.com/en-us/azure/azure-netapp-files/azure-netapp-files-set-up-capacity-pool) for more information.
```
Comment (WodansSon) [inline website/docs/r/netapp_pool.html.markdown:53]: We should not include the locale (e.g., `en-us`) in the product link and allow the sites locale picker determine which locale the documentation should be served in.
```suggestion
~> **NOTE** `2` TB capacity pool sizing is currently in preview. You can only take advantage of the `2` TB minimum if all the volumes in the capacity pool are using `Standard` network features. If any volume is using `Basic` network features, the minimum size is `4` TB. Please see the product [documentation](https://learn.microsoft.com/azure/azure-netapp-files/azure-netapp-files-set-up-capacity-pool) for more information.
```

**Review by WodansSon — CHANGES_REQUESTED** [review]
@neil-yechenwei, I had a look as well and left a single comment about the product link including the local of the documentation. If you could take a look that would be 🔥

File: website/docs/r/netapp_pool.html.markdown (hunk 2 above)
Comment (neil-yechenwei) [reply website/docs/r/netapp_pool.html.markdown:53]: Thanks. Updated.

**Review by stephybun — APPROVED** [review]
Thanks @neil-yechenwei LGTM 👍

**Pull request merged**